content/en/docs/tasks/traffic-management/locality-load-balancing/before-you-begin/index.md

```diff
@@ -36,10 +36,12 @@ The following environment variables will be used for the various contexts:
 Variable | Description
 -------- | -----------
 `CTX_PRIMARY` | The context used for applying configuration to the primary cluster.
-`CTX_R1_Z1` | The context used to interact with pods in `region1.zone1`.
-`CTX_R1_Z2` | The context used to interact with pods in `region1.zone2`.
-`CTX_R2_Z3` | The context used to interact with pods in `region2.zone3`.
-`CTX_R3_Z4` | The context used to interact with pods in `region3.zone4`.
+`CTX_WORKERS` | The contexts used to interact with pods in the worker clusters.
```

Comment:
Using an array and relying on index positions seems generally less clear IMO. I think we should just revert this